frontend/public/help/routeMap.json

```diff
@@ -1,6 +1,54 @@
 {
+  "auth": "",
```

Comment:
Would it be better to use an array instead of a map for more configurable? For example: 
```
[
  {
    routeName: "",
    helpName: "",
    // some configurable field in the future.
  }
]
```